index.html

```diff
@@ -93,7 +93,6 @@
       import { Buffer } from "buffer";
       window.Buffer = Buffer;
     </script>
-    <title>Radicle Interface</title>
```

Comment:
I’d keep this and set it to “Radicle”. This way you’ll have a title until all the Javascript is loaded and the app has booted.

Reply:
2b7afe4917f870515910b624295a725559da4e48